manifests/d/DGP-Studio/Snap-Hutao/1.10.7/DGP-Studio.Snap-Hutao.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: DGP-Studio.Snap-Hutao
+PackageVersion: 1.10.7
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.19041.0
+InstallerType: msix
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+PackageFamilyName: 60568DGPStudio.SnapHutao_wbnnev551gwxy
+RestrictedCapabilities:
+- runFullTrust
+- unvirtualizedResources
```

Comment:
@stephengillie - Is it related to `RestrictedCapabilities`? There're `runFullTrust` and `unvirtualizedResources` in it.

Maybe we can't verify this package in _Windows Sandbox_ or _VM instances_?

Reply:
See https://github.com/microsoft/winget-pkgs/pull/164380#issuecomment-2249535576.

Appx environment is partially supported by _Windows Sandbox_, so it might not pass the Azure Pipeline and manual VM validation.

@stephengillie - Shall I close the PR? 🙏🏼